content/03-pytest.md

```diff
@@ -117,27 +117,57 @@ whether this works on Windows):
    $ git add example.py
    $ git commit
    ```
-5. Git add and commit the file:
-   ```shell
-   $ git add example.py
-   $ git commit
-   ```
-6. Create the following shell script. Save this file as `pre-commit`:
+5. Create the following shell script. Save this file as `pre-commit`:
    ```shell
    #!/bin/bash
 
    pytest example.py
    ```
-7. Make `pre-commit` executable:
-   ```shell
-   $ chmod +x pre-commit
-   ```
+6. Make `pre-commit` executable:
+      `````{tabs}
+         ````{tab} Bash
+     
+            ```bash
+            $ chmod +x pre-commit
+            ```
+         ````
+         ````{tab} Anaconda prompt
```

Comment:
```suggestion
         ````{group-tab} Windows Anaconda prompt
```
1) group-tab instead
2) is this Windows specific?  Is there an Anaconda prompt on other OSs ?

Reply:
First I thought to make a Linux/macOS tab and a Windows tab but since there are at least 3-4 ways in Windows, I changed that.